Index.html

```diff
@@ -11,318 +11,355 @@
+					/>
+					<textarea
+						name="message"
+						id="message"
+						class="formEl"
+						cols="30"
+						rows="10"
+						maxlength="500"
+						placeholder="Please leave your message here"
+						required
+					></textarea>
+					<input class="projectButton btn" type="submit" value="See Project" />
+				</form>
 			</div>
-			<form action="https://formspree.io/f/meqnjrwb" method="post">
-				<input
-					name="name"
-					id="name"
-					class="formEl"
-					type="text"
-					placeholder="Your name"
-					maxlength="30"
-					required
-				/>
-				<input
-					name="email"
-					id="email"
-					class="formEl"
-					type="email"
-					placeholder="Your email address"
-					required
-				/>
-				<textarea
-					name="message"
-					id="message"
-					class="formEl"
-					cols="30"
-					rows="10"
-					maxlength="500"
-					placeholder="Please leave your message here"
-					required
-				></textarea>
-				<input class="projectButton btn" type="submit" value="See Project" />
-			</form>
-			<img class="line" src="./assets/vectorLine.svg" alt="horizontal line" />
-			<div class="socials">
-				<ul class="profiles footer-icons">
-					<li>
-						<a href="#">
-							<img class="icons" src="assets/github.svg" alt="GitHub Logo" />
-						</a>
-					</li>
-					<li>
-						<a href="#">
-							<img
-								class="icons"
-								src="assets/linkedInn.png"
-								alt="LinkedIn Logo"
-							/>
-						</a>
-					</li>
-					<li>
-						<a href="#">
-							<img
-								class="icons"
-								src="assets/angellist.svg"
-								alt="Angellist Logo"
-							/>
-						</a>
-					</li>
-					<li>
-						<a href="#">
-							<img class="icons" src="assets/Twitter.svg" alt="Twitter Logo" />
-						</a>
-					</li>
-					<li>
-						<a href="#">
-							<img class="icons" src="assets/medium.svg" alt="Medium Logo" />
-						</a>
-					</li>
-				</ul>
+			<img class="line4" src="./assets/vectorLine.svg" alt="horizontal line" />
+			<img class="desktopLine4" src="./assets/desktopLine4.svg" alt="horizontal line" />
+			<div class="socialsAndEmail">
+				<div class="socials footer-socials">
+					<ul class="profiles footer-icons">
+						<li>
+							<a href="#">
+								<img class="icons" src="assets/github.svg" alt="GitHub Logo" />
+							</a>
+						</li>
+						<li>
+							<a href="#">
+								<img
+									class="icons"
+									src="assets/linkedInn.png"
+									alt="LinkedIn Logo"
+								/>
+							</a>
+						</li>
+						<li>
+							<a href="#">
+								<img
+									class="icons"
+									src="assets/angellist.svg"
+									alt="Angellist Logo"
+								/>
+							</a>
+						</li>
+						<li>
+							<a href="#">
+								<img class="icons" src="assets/Twitter.svg" alt="Twitter Logo" />
+							</a>
+						</li>
+						<li>
+							<a href="#">
+								<img class="icons" src="assets/medium.svg" alt="Medium Logo" />
+							</a>
+						</li>
+					</ul>
+				</div>
+				<span class="mail">emailhere@email.com</span>
```

Comment:
- [ ] [OPTIONAL] I think it could be a good idea to add the `mailto` attribute so the users can send you an email when clicking the link. You can see a guide [here](https://css-tricks.com/snippets/html/mailto-links/) :wink: